packages/vue/package.json

```diff
@@ -57,7 +58,9 @@
   "devDependencies": {
     "@testing-library/vue": "^8.1.0",
     "@vue.ts/tsx-auto-props": "^0.6.0",
-    "vue": "3.5.12"
+    "unplugin-vue": "^5.2.1",
+    "vue": "3.5.12",
+    "vue-tsc": "^2.0.24"
```

Comment:
`unplugin-vue` - esbuild plugin to tranform `.vue` files to JS
`vue-tsc` - dts emit with `.vue` file support